doc-content/jbpm-docs/src/main/asciidoc/BAM/DashbuilderRuntime-section.adoc

```diff
@@ -16,29 +16,128 @@ After login, users may upload dashboard files exported from {CENTRAL}
 image::BAM/DashbuilderRuntimeWelcomePage.png[]
 
 == Exporting Dashbuilder data
-You can export the dashbuilder related data such as datasets, pages, and navigation from {CENTRAL} as a ZIP file. If the exported dashboard contains only one page then it will be used as the default page or you can select a default page by just naming it *index*.
+
+You can export the dashbuilder related data such as datasets, pages, and navigation from {CENTRAL} as a ZIP file.
+
+.Procedure
+. In {CENTRAL}, select the *Admin* icon in the top-right corner of the screen and select *Dashbuilder Data Transfer*.
+
+. To export the dashbuilder related data, do any of the following tasks:
++
+--
+If you want to export all dashbuilder data as a ZIP file, do the following tasks:
+
+... On the *Dashbuilder Data Transfer* page, click *Export all*.
++
+An `export.zip` file containing all dashbuilder data is downloaded. The `export.zip` file structure is separated by data type. For example:
++
+[source]
+----
+dashbuilder/datasets/definitions/dataset-example1.dset
+dashbuilder/datasets/definitions/dataset-example2.dset
+dashbuilder/datasets/readme.md
+dashbuilder/perspectives/page1/perspective_layout
+dashbuilder/perspectives/page1/perspective_layout.plugin
+dashbuilder/perspectives/page2/perspective_layout
+dashbuilder/perspectives/page2/perspective_layout.plugin
+dashbuilder/perspectives/readme.md
+dashbuilder/navigation/navigation/navtree.json
+dashbuilder/navigation/readme.md
+VERSION
+----
+
+
+If you want to export customized user created dashbuilder data as a ZIP file, do the following tasks:
```

Comment:
Just one note here - in the "Export all" option, it's also user created data that is exported (user created + provided). Just saying this, because it could be a bit misleading for the user.

Reply:
Done